models/app.model.js

```diff
@@ -7,7 +7,28 @@ exports.selectTopics = () => {
   });
 };
 
-exports.selectArticle = (article_id) => {
+exports.selectArticle = (topic) => {
+  if (topic) {
+    return db
```

Comment:
We can make this more DRY by storing the query statement in a variable, and just adding to the string dependent on the topic being present or not BEFORE making any query requests to the database. We don't really want tobe making multiple queries to the database if we can help it. This will be a much more preferable way of doing queries especially once we start to add in more query and sort by options. for example:

let queryString = `SELECT STUFF HERE....`

if (topic) {
queryString += `WHERE ....`
}

queryString += `GROUP BY....`

return db.query(queryString)